docker/kubernetes/templates/tubemq-broker-ini-configmap.yaml

```
@@ -26,7 +26,7 @@ data:
   broker.ini.raw: |
     [broker]
     brokerId=REPLACE_BY_POD_ID
-    hostName=REPLACE_BY_POD_HOSTNAME
+    hostName=REPLACE_BY_POD_IP
     port=8123
     webPort=8081
```

@dockerzhang These ports are configurable. It should not be hard-coded.

```suggestion
    port={{ .Values.tubemqBroker.ports.rpcPort }}
    webPort={{ .Values.tubemqBroker.ports.webPort }}
```